### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.8)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#29362]
#### VFS team name
Education Application (BAH)

#### VFS product name
Comparison Tool Redesign

#### Point of Contact/Reviewers
Allison Christman (Design)

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
The H1 above the tabs is center aligned

### Link, screenshot or steps to recreate
https://staging.va.gov/gi-bill-comparison-tool-sandbox/?search=name&name=cincinnati

### VA.gov Experience Standard
[Cateogy Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
This should be left aligned

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by amyknox** [comment]
@dneel-bah 

[Desktop prototype here](https://bahdigital.invisionapp.com/share/QCIAGXOTD3P#/296062999_Searchbyname_Desktop_4). 
[Mobile prototype here](https://bahdigital.invisionapp.com/share/QDIAGYESX4T#/296060491_CT_Searchbyname_mobile_1)

The prototype update also addresses 29362 (H1 should be left-aligned, not centered on the page)

**Comment by elder-joseph** [comment]
Associated PR: https://github.com/department-of-veterans-affairs/vets-website/pull/18504
